src/softwareHeritage/agent/softwareHeritageAgent.php

```diff
@@ -162,16 +163,21 @@ protected function getSoftwareHeritageLicense($sha256)
     $URI = $this->configuration['api']['url'].$this->configuration['api']['uri'].$sha256.$this->configuration['api']['content'];
     $response = $client->get($URI);
     $statusCode = $response->getStatusCode();
-    if (200 === $statusCode) {
-      $responseContent = json_decode($response->getBody()->getContents(),true);
-      $licenseRecord = $responseContent["facts"][0]["licenses"];
-      return $licenseRecord;
-    } else if (429 === $statusCode) {
-      $responseContent = $response->getHeaders();
-      return $responseContent["X-RateLimit-Reset"][0];
-    } else {
-      return [];
-    }
+    $cookedResult = array();
+    switch ($statusCode) :
```

Comment:
This `switch/case` syntax is not familiar to me. Can you please provide an reference to this syntax?